pkg/app/app.go

```diff
@@ -167,6 +173,13 @@ func RealMain(opts types.Options, d types.Deployer, tester types.Tester) (result
 			testErr = test.Run()
 		}
 
+		if opts.UploadRundir() {
+			cmd := exec.Command("cp", "-r", artifacts.RunDir(), artifacts.BaseDir())
```

Comment:
// TODO: copy in golang?

Worth noting that actually mimicing cp -r correctly takes some effort.
KIND has a OK implementation at https://github.com/kubernetes-sigs/kind/blob/de5b3e615a4f8b03d2f525e9faf397b12f8a2339/pkg/fs/fs.go#L57

Reply:
That's not package meant be imported outside that project, it's better to copy here the logic.

Reply:
There is already an fs package in `kubetest2`. Copied missing functions for directory copy from `kind` repo.